example/intercept/intercept.cc

```diff
@@ -538,9 +538,9 @@ TSPluginInit(int /* argc */, const char * /* argv */ [])
 {
   TSPluginRegistrationInfo info;
 
-  info.plugin_name   = (char *)PLUGIN;
-  info.vendor_name   = (char *)"MyCompany";
-  info.support_email = (char *)"ts-api-support@MyCompany.com";
+  info.plugin_name   = PLUGIN;
+  info.vendor_name   = "MyCompany";
+  info.support_email = "ts-api-support@MyCompany.com";
```

Comment:
Since you are changing all these, what do you think about standardizing on:
```C
info.plugin_name   = PLUGIN_NAME;
info.vendor_name   = "Apache Software Foundation";
info.support_email = "dev@trafficserver.apache.org";
```